File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let sum = 0;
    let product = 1;
    
    for (let i = 0; i < num_list.length; i++) {
        sum += num_list[i];
        product *= num_list[i];
    }
    if (product < sum * sum) return 1;
    else return 0;
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    total = 0
    product = 1
    
    for i in range(len(num_list)):
        total += num_list[i]
        product *= num_list[i]
        
    if product < total * total:
        return 1
    else:
        return 0